net.sourceforge.pdfchain.json

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "net.sourceforge.pdfchain",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
+    "command": "pdfchain",
+    "finish-args": [
+    	"--filesystem=xdg-documents",
+        "--filesystem=xdg-download",
+        "--filesystem=xdg-pictures",
+        "--filesystem=home:ro",
+        "--filesystem=host:ro",
+        "--share=ipc",
+        "--socket=x11",
```

Comment:
```suggestion
        "--socket=fallback-x11",
```

Reply:
Ok.

Reply:
actually the bigger question here is "does it support wayland".
Given that you build `xgetres` which make me think it would fail with just wayland.

Reply:
With "--socket=fallback-x11" the pdftk java app fails with "cannot open display" and indeed I opened a terminal and the $DISPLAY env var is empty. With "--socket=x11" it works fine.